app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -173,13 +173,7 @@ def x509_session?
   end
 
   def x509_presented
```

Comment:
since we're here, and this definitely returns a boolean now, WDYT about renaming to have a `?` (also good for a follow-up PR if we want to keep this one small)
```suggestion
  def x509_presented?
```

Reply:
oh yeah actually ... i could just remove this method now and inline it since it's so smol!

Reply:
moved in https://github.com/18F/identity-idp/pull/10701/commits/27e5a37784c3cd54b6f4deaf01a81d5b4df10fc5